app/lib/katello/errors.rb

```diff
@@ -35,6 +35,12 @@ def message
       end
     end
 
+    class EmptyBulkActionException < StandardError
+      def message
+        _("No registered Hosts found in selection.")
```

Comment:
User might be confused, since it's "registered' just not for content.  Can we match the other text? Like `No hosts with katello-agent` or `No hosts with content` or something.